src/test/app/drafts/models/claimant.ts

```diff
@@ -1,22 +1,20 @@
 import { expect } from 'chai'
 
 import Claimant from 'drafts/models/claimant'
-import { PartyDetails } from 'forms/models/partyDetails'
 
 /* Allow chai assertions which don't end in a function call, e.g. expect(thing).to.be.undefined */
 /* tslint:disable:no-unused-expression */
 describe('Claimant', () => {
   describe('constructor', () => {
     it('should have undefined name, dateOfBirth, mobilePhone fields', () => {
       let claimant = new Claimant()
-      expect(claimant.name).to.be.undefined
-      expect(claimant.dateOfBirth).to.be.undefined
+      expect(claimant.partyDetails).to.be.undefined
       expect(claimant.mobilePhone).to.be.undefined
     })
 
     it('should have party details field initialised', () => {
       let claimant = new Claimant()
-      expect(claimant.partyDetails).to.be.instanceOf(PartyDetails)
+      expect(claimant.partyDetails).to.be.eq(undefined)
```

Comment:
Change in test does not reflect previous test expectation judging by test description. Please fix.